### DISCUSSION TITLE
Documentation and Community

### DISCUSSION THREAD

**Discussion by markeaston63** [hugerte/hugerte#9] [General]
What we need is two things:

1. Documentation on HugeMCE and lots of it. Googing for info about how to use TinyMCE 6.8.3 is difficult as so much info relates to other versions. In using HugeMCE I have had to sift through all that as I resolved issues. ANd a lot of misinformation as well. I am willing to make a start on this documentation as I am doing it internally at my company anyway. So I may as well do it here.
2. We (Naverisk) are committed to HugeMCE and so will be willing to be active in this community and with resolving issues. We need an active community. Are peope interested? HugeMCE is a good thing,.

**Comment by carlosmintfan** [comment]
Locked in favour of https://github.com/hugemce/discussions/discussions/5. I made a mistake in dedicating an own repo to discussions, but changing the organization default to this repo would break (change) links, which I don‘t want. Links to existing discussions would redirect to different issues…I‘ll see I disable discussions for this repo.
